abdul-moize/django_ecommerce
# Feat: Docker Files

This PR adds docker files to main branch. The two important files are  `Dockerfile` & `nginx.default`. Dockerfile contains instructions to set up the container environment. nginx.default contains server routes and mappings for static files.

To test this PR follow these steps:
1. Pull this branch
2. Install docker by following this [guide](https://docs.docker.com/engine/install/ubuntu/). Ensure you follow the post installation steps given [here](https://docs.docker.com/engine/install/linux-postinstall/)
3. Run : `docker build . -t webapp:latest`
4. Run : `docker run -p 8000:8000/tcp webapp:latest`
5. Open `127.0.0.1:8000` in web browser
6. Login with email `admin@admin.com` password `adminpassword`. If you want create a different super user you can do so by attaching a shell to running container. First you need to get container id which you get by `docker ps`. Afterwards run `docker ecec -it container_id bash`
7. Add products and test the app

Please let me know if there are any concerns, problems & comments
